modules/sdk-coin-eth/src/lib/index.ts

```diff
@@ -0,0 +1,17 @@
+export * from './contractCall';
+export * from './iface';
+export * from './keyPair';
+export * from './transaction';
+export * from './transactionBuilder';
+export * from './transferBuilder';
+export * from './transferBuilders';
+export * from './transferBuilder';
+export * from './types';
+export * from './utils';
+export * from './walletUtil';
+
+// for Backwards Compatibility
```

Comment:
nit: Would it be helpful to add:
```
/**
 * @deprecated
 */
```
To find all them faster?

Reply:
don't think that would do that much, since these are just re-exports. We don't want to flag the underlying types either as deprecated